pandera/backends/pyspark/decorators.py

```diff
@@ -126,3 +131,54 @@ def _get_check_obj():
         return wrapper
 
     return _wrapper
+
+
+def cache_check_obj():
+    """This decorator evaluates if `check_obj` can be cached before validation.
+
+    As a new Spark action is triggered for every new data check added to the Pandera
+     schema by the user, Spark keeps reprocessing the `check_obj` dataframe lots of
```

Comment:
Sorry for being extremly picky here, but is this extra space intentional on the longer part of the docstring?

Reply:
Changed, tks